packages/fetch-and-compile/README.md

```diff
@@ -65,7 +65,7 @@ async function decode(address: string) {
   console.groupEnd();
   console.log(")");
 }
-// 🥳 Try it out with a contract.
+// 🥳 Try it out with a contract - this is the address for the ENS registry
```

Comment:
I think it would be helpful to include what's expected here:

```console
setResolver(
  node: 0xf96f66c6c5e18dc3da0e5d238ee5ff0f56ad7876717492cfcbb3421db607e44c
  resolver: 0x4976fb03C32e5B8cfe2b6cCB31c09Ba78EBaBa41
)
```

Reply:
what do you mean? is that return value that `decode` gives?

Reply:
Yes, that's the output `decode()` logs when for the given address.

Reply:
cool, I added a bit about the output. that work?